### PR TITLE
feat(Button component storybook): Updates the button componets story book to …

### PR DESCRIPTION
**Pull request by Fodilahy-mena**
Hello team,

I have created a new task for updating the button component storybook because of new changes from our figma design. 
Now it's done and ready to be reviewed. Please have a look and give some suggestion where needed!

Click up link: https://app.clickup.com/t/4djjud
Refs: part of U2 #4djjud

Thanks in advance!

### CODE REVIEW

**Review by starjardin — APPROVED** [review]

**Review by vahona — APPROVED** [review]

File: src/components/Buttons/Buttons.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react'
 import styled from 'styled-components'
 import { fonts } from '../../globalStyles/fonts'
+import PendingIndicator from '../PendingIndicator/PendingIndicator'
+import GoogleIcon from '../../assets/google.svg';
+import { mediaQueriesPx } from "../../mediaQueries";
+
 export interface ButtonsProps {
   label?: string
   learnEnabled?: boolean
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx:10]: rename leasrnEnabled to isPrimary

File: src/components/Buttons/Buttons.tsx [hunk 2]
```diff
@@ -16,47 +21,66 @@ const FirstWrapper = styled.div`
...
+    height: 18px;
+    ${mediaQueriesPx('md', 'xl')`
+      width: 20px;
+      height: 20px;
+    `}
+  }
 `
+const GoogleImage = styled.img``;
+
 const Buttons: React.FC<ButtonsProps> = ({
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx:78]: Rename Buttons to Button

File: src/components/Buttons/Buttons.tsx [hunk 3]
```diff
@@ -16,47 +21,66 @@ const FirstWrapper = styled.div`
...
   label,
   onClick,
   disabled,
   learnEnabled,
+  buttonName,
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx:83]: this should be just isLoading of type boolean

File: src/components/Buttons/Buttons.tsx [hunk 4]
```diff
@@ -68,7 +92,9 @@ const Buttons: React.FC<ButtonsProps> = ({
           disabled={disabled}
           onClick={() => ''}
           {...props}>
-          {label}
+            {buttonName === 'loading' ?
+            <PendingIndicator size='xs' alt="loading icon"/> : buttonName === 'autoformatwithicon' ? <GoogleImage src={GoogleIcon} alt="google icon"/> : ''}
+          <span>{label}</span>
```
Comment (andrzej-onja) [inline src/components/Buttons/Buttons.tsx:97]: google image should be passed as a prop, If present display it if no just span

File: src/components/Buttons/Buttons.tsx [hunk 5]
```diff
@@ -16,47 +21,66 @@ const FirstWrapper = styled.div`
 const SecondWrapper = styled.div`
   display: inline-block;
   background: rgba(21, 140, 177, 0.3);
-  position: absolute;
+  /* position: absolute;
   top: 6px;
```
[the remark this answers is not in the document]
Comment (Fodilahy-mena) [reply src/components/Buttons/Buttons.tsx:25]: No, thanks

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**